index.js

```diff
@@ -46,15 +46,44 @@ class CoverageBase {
     /**
      * Get shell command to upload coverage to server
      * @method getUploadCoverageCmd
-     * @return {Promise}     Shell commands to upload coverage
+     * @param   {Object}  config
+     * @param   {String}  config.build   build configuration
+     * @return {Promise}  Shell commands to upload coverage
      */
-    getUploadCoverageCmd() {
+    getUploadCoverageCmd(config) {
+        if (this.isCoverageEnabled(this.config, config.build) === 'false') {
+            const skipMessage = 'Coverage feature is skipped. ' +
+                'Set SD_COVERAGE_PLUGIN_ENABLED environment true, if you want to get coverages.';
```

Comment:
I think this msg should be 
environment -> environment variable 
or
SD_COVERAGE_PLUGIN_ENABLED environment -> $SD_COVERAGE_PLUGIN_ENABLED

Reply:
I fixed to $SD_COVERAGE_PLUGIN_ENABLED.